### ISSUE TITLE
Bad detection of acronyms into Location.URL

### ISSUE THREAD

**Issue by benlabbe** [aymara/lima#96]
**Describe the bug**
The acronyms with dot separators are wrongly recognized as Location.URL NamedEntities.

The ``RegexMatcher`` in ``lima-lp-fre.xml`` seams to match acronyms into t_url tokens, for which the \`LOCATION-fre.rules`` rule file has a direct assignation into Location.URL.

**To Reproduce**
```sh
$ echo -e "Je fais partie du service S.I.A.L.V. . Constulstez l'adresse www.kalisteo.fr pour plus de détails sur nos activités" >tmp
$ analyzeText -l fre tmp
Analyzing 1/1 (100.00%) 'tmp'
# global.columns = = ID	FORM	LEMMA	UPOS	XPOS	FEATS	HEAD	DEPREL	DEPS	MISC
# sent_id = 1
# text = Je fais partie du service S.I.A.L.V.
1	Je	je	CLS	_	_	2	PronSujVerbe	_	Pos=1|Len=2
2	fais partie	faire partie	V	_	_	_	_	_	Pos=4|Len=11
3	du	du	P+D	_	_	4	PREPSUB	_	Pos=16|Len=2
4	service	service	NC	_	_	2	CPL_V	_	Pos=19|Len=7
5	S.I.A.L.V		NONE	_	_	_	_	_	NE=Location.URL|Pos=27|Len=9
6	.	.	PONCTU_FORTE	_	_	_	_	_	Pos=38|Len=1

# sent_id = 2
# text = Constulstez l'adresse www.kalisteo.fr pour plus de détails sur nos activités 
1	Constulstez	Constulstez	NPP	_	_	_	_	_	Pos=40|Len=11
2	l'	le	DET	_	_	_	_	_	Pos=52|Len=2|SpaceAfter=No
3	adresse	adresse	NC	_	_	_	_	_	Pos=54|Len=7
4	www.kalisteo.fr		NONE	_	_	_	_	_	NE=Location.URL|Pos=62|Len=15
5	pour	pour	P	_	_	8	PREPSUB	_	Pos=78|Len=4
6	plus	plus	ADV	_	_	4	AdvSub	_	Pos=83|Len=4
7	de	de	P	_	_	8	PREPSUB	_	Pos=88|Len=2
8	détails	détail	NC	_	_	3	COMPDUNOM	_	Pos=91|Len=7
9	sur	sur	P	_	_	11	PREPSUB	_	Pos=99|Len=3
10	nos	nos	DET	_	_	_	_	_	Pos=103|Len=3
11	activités	activité	NC	_	_	8	COMPDUNOM	_	Pos=107|Len=9
```

**Expected behavior**
The `S.I.A.L.V.` acronym should be detected as a t_acronym token (and then eventually as Organisation NamedEntity)

The ``RegexMatcher`` in ``lima-lp-fre.xml`` could have a more specific rule to match as t_acronym only the dot separated acronyms.

The rule should identify those acronyms by the fact that the alpha numeric parts of the acronyms are only one character long.

Tokens with multi char parts between the dot separators could then be matched into t_url tokens.

**Version**
```
commit ce9d5525474f7b14b2ebfdb3266775e6d3d28cdc (HEAD -> master, origin/master, origin/HEAD)
Author: Gael de Chalendar <gael.de-chalendar@cea.fr>
Date:   Wed Dec 11 10:14:54 2019 +0100
```

**Comment by benlabbe** [comment]
This is fixed since commit 565617d2 which fix some of the regexp in the regepx declaration of lima-lp-(fre|eng).xml

**Comment by benlabbe** [comment]
I restored some rules inside LOCATION_COMP-fre.rules because the buildbot stopped on a failling tva test.
